### PR TITLE
PTW-61: Update to fix sort order filter - also consolidates all filte…

### PR DESCRIPTION
**Pull request by peterchallett**
…rs to one annotation per type.

### CODE REVIEW

File: lib/MwbExporter/Formatter/Doctrine2/Annotation/OnCorps/ApiPlatformFieldAnnotations.php [hunk 1]
```diff
@@ -10,18 +10,24 @@
...
-
     /**
      * @param Model\Table $table
+     * @param string      $fieldName
      *
-     * @return $this
+     * @return string
      */
-    abstract function buildAnnotations(Model\Table $table);
+    abstract function buildAnnotationProperty(Model\Table $table, Model\Column $column, string $type): string;
```
Comment (domgraziano) [inline lib/MwbExporter/Formatter/Doctrine2/Annotation/OnCorps/ApiPlatformFieldAnnotations.php:25]: for better readability can I suggest we use a `use` statement at the top of the file for ...`Model\Table` so we can use `Table` ?

**Review by domgraziano — COMMENTED** [review]
@peterchallett please see https://github.com/OnCorps/doctrine2-exporter/commit/6d62b2654e7c18e33be9c387508ddea31be008a4#r34444544 as I think we've accidentally deleted a use statement important for uuid usage in previous PR,
can we please restore that code so I'd be able to test this too ?

code there was

```

        if($this->getConfig()->get(Formatter::CFG_RASMEY_UUID_PROVIDER)) {
            $uses[] = 'Ramsey\Uuid\Uuid';
        }

```

File: lib/MwbExporter/Formatter/Doctrine2/Annotation/OnCorps/ApiPlatformFieldAnnotations.php (hunk 1 above)
Comment (peterchallett) [reply lib/MwbExporter/Formatter/Doctrine2/Annotation/OnCorps/ApiPlatformFieldAnnotations.php:25]: I'd like to keep this as is please.

Comment (domgraziano) [reply lib/MwbExporter/Formatter/Doctrine2/Annotation/OnCorps/ApiPlatformFieldAnnotations.php:25]: ok, just mentioning that the suggested usage of use statement here is a common standardised pattern in the wider php community and it is something that we actively flagged in other OnCorps php codebase as I think it is part of a psr coding style standard, I think phpstorm auto-import in fact works based on that.

**Review by domgraziano — APPROVED** [review]

**Pull request merged**